tools/fruit_generator.rb

```diff
@@ -1,24 +1,36 @@
+# -----------------------------------------------------------------------------
```

Comment:
Just seeing whether it will work on a re-run...

Reply:
It does not look like

Reply:
Ah, I understand! In Makefile line 86, we replace the contents of line 4 in this file on the fly, in order to point at the correct `rake_base.rb` file. That needs to change to be line 17 now. Could you do that, rather than me? Just change the `sed -i "4s,.*` to `sed -i "17s,.*.